packages/analytics-js-integrations/src/integrations/GoogleAds/utils.js

```diff
@@ -114,4 +114,20 @@ function getConversionData(eventTypeConversions, eventName, defaultPageConversio
   return conversionData;
 }
 
-export { shouldSendConversionEvent, shouldSendDynamicRemarketingEvent, getConversionData };
+function newCustomerAcquisitionReporting(properties, newCustomerFromCOnfig) {
+  const updatedProperties = { ...properties }; // create a copy of properties object
+  if (updatedProperties && updatedProperties.newCustomer) {
+    updatedProperties.new_customer = updatedProperties.newCustomer;
+    delete updatedProperties.newCustomer;
+  } else if (newCustomerFromCOnfig !== 'unspecified') {
```

Comment:
```suggestion
  } else if (newCustomerFromCOnfig !== undefined) {
```

Reply:
I made it unspecified according to [here](https://support.google.com/google-ads/answer/12077475?hl=en#zippy=%2Cinstall-with-the-global-site-tag%2Cinstall-with-google-tag-manager:~:text=Step%202%3A%20Add%20the%20new_customer%20parameter%20to%20your%20tag%20event%20snippet), but we can keep it undefined too as suggested. Addressed.